internal/pkg/utils/testproject/project.go

```diff
@@ -173,6 +183,23 @@ func (p *Project) Clear() {
 		assert.FailNow(p.t, fmt.Sprintf("cannot delete branches: %s", err))
 	}
 
+	// Delete metadata of default branch
+	pool = p.storageApi.NewPool()
+	pool.Request(p.storageApi.ListBranchMetadataRequest(p.defaultBranch.Id)).
+		OnSuccess(func(response *client.Response) {
+			branchMetadataResponse := *response.Result().(*[]storageapi.Metadata)
+			for _, m := range branchMetadataResponse {
+				pool.Request(p.storageApi.NewRequest(resty.MethodDelete, "branch/{branchId}/metadata/{metadataId}").
+					SetPathParam("branchId", p.defaultBranch.Id.String()).
+					SetPathParam("metadataId", m.Id).
+					Send())
```

Comment:
Toto by sa mohlo hodit do `storageapi` balicka, aj ked sa to pouziva iba v testoch.
V dlhodobom plane je plnohodnotny Storage API klient v Go, tak aby sa to nemuselo potom hladat.

Reply:
(IMHO nevadi, ze tam nie je delete pre config metadata, dorobi sa ked to bude potrebne)

Reply:
Tak jsem to zkusil (https://github.com/keboola/keboola-as-code/pull/642/commits/e93f55434b21077eddf6c30198d198eaebd7105d), ale nedaří se mi zprovoznit test. 🙈

Netušíš v čem je problém? Vezmu si id vylistovanýho metadata z předchozího testu a pošlu na něj delete a test mi tvrdí že neexistuje:

![CleanShot 2022-04-25 at 17 41 53](https://user-images.githubusercontent.com/215660/165124601-b0bcc798-6119-44ba-b554-7d39e3ed469c.jpg)


![CleanShot 2022-04-25 at 17 40 30](https://user-images.githubusercontent.com/215660/165124216-f6b9106f-f923-407b-8d40-7d012e3f36b3.jpg)

Reply:
Idem to pozriet ...

Reply:
@JakubMatejka posielas request 2x a spadne ten 2.:

Odtial vyhod send:
https://github.com/keboola/keboola-as-code/blob/e93f55434b21077eddf6c30198d198eaebd7105d/internal/pkg/api/client/storageapi/branch.go#L187

A chyba je aj tu, predtym som si to nevsimol:
https://github.com/keboola/keboola-as-code/blob/e93f55434b21077eddf6c30198d198eaebd7105d/internal/pkg/utils/testproject/project.go#L191

Teda ^^^ tu `Send` zostane, ale najprv zaregistruj request do `pool`, a z tak volaj `Send`.
`				pool.
					Request(p.storageApi.DeleteBranchMetadataRequest(p.defaultBranch.Id, m.Id)).
					Send()
`

(pridam tam kontrolu, aby sa uz odoslany request nedal zaregistrovat do pool)

Reply:
omg, taková zjevnost 🙈 děkuju! A jo vidíš, tak ta druhá věc by mi asi jen tak nedošla. 😅

Reply:
Jj, nebolo to osetrene, fixnute tu:
https://github.com/keboola/keboola-as-code/pull/645